crates/bevy_gltf/src/loader.rs

```diff
@@ -697,7 +697,7 @@ fn load_node(
     gltf_node: &gltf::Node,
     world_builder: &mut WorldChildBuilder,
     load_context: &mut LoadContext,
-    buffer_data: &[Vec<u8>],
+    _buffer_data: &[Vec<u8>],
```

Comment:
if this is unused in this function, you can completely remove it

Reply:
The warning was "only used in recursion."  Yeah, it's surprising no one's interested in `buffer_data` for a node in GLTFs; I guess `load_node` doesn't care; other methods use that.  I'll take it out.